scripts/src/public/services/users.ts

```diff
@@ -139,8 +141,24 @@ export async function activateUser(authToken: DbAuthToken, user: User): Promise<
 }
 
 export async function userExists(appId: string, appUserId: string): Promise<boolean> {
+	// get from cache
+	const redis = getRedisClient();
+	const key = `app:${ appId }:user:${ appUserId }:exists`;
+	const value = await redis.async.get(key);
+	if (value) {
+		return JSON.parse(value);
```

Comment:
the function returns `Promise<boolean>`, but you return here the value.
shouldn't it just be `return true`?

Reply:
It's ok the function is `async` and this is syntactic sugar.
I'm also caching `false` responses but only for 15 minutes

Reply:
@doodyparizada 
what does the async part have to do with this?
my point is that `JSON.parse(value)` won't return a boolean, so it's not `Promise<boolean>` but `Promise<User>`, which is wrong according to the function signature.